keps/sig-node/2411-cri-container-log-rotation/kep.yaml

```diff
@@ -0,0 +1,46 @@
+title: CRI Container Log Rotation
+kep-number: 2411
+authors:
+  - "@umohnani8"
+owning-sig: sig-node
+participating-sigs:
+  - sig-instrumentation
+  - sig-storage
+status: implemented
+creation-date: 2018-01-25
+reviewers:
+  - "@mrunalp"
+  - "@SergeyKanzhelev"
+approvers:
+  - "@mrunalp"
+  - "@SergeyKanzhelev"
```

Comment:
you can keep me just in reviewers as I'n not an approver for the SIG. Thank you!

Reply:
Fixed, thanks for the review!